wasmtime-environ/src/cache.rs

```diff
@@ -505,3 +505,343 @@ impl<'de> Deserialize<'de> for JtOffsetsWrapper<'_> {
         deserializer.deserialize_seq(JtOffsetsWrapperVisitor {})
     }
 }
+
+//-////////////////////////////////////////////////////////////////////
+//-//////////////////////////// Tests /////////////////////////////////
+//-////////////////////////////////////////////////////////////////////
+
+// Since cache system is a global thing, each test needs to be run in seperate process.
+// So, init() tests are run as integration tests.
+// However, caching is a private thing and needs to be tested internally. Thus we have
+// one big test here.
```

Comment:
This is a pretty big test; would it make sense to move it to a separate file in the `wasmtime-environ/tests` directory? Or is this comment saying that it's using things that aren't visible outside this file?

Reply:
We can't load nor store cache outside the `wasmtime-environ`. The idea is that it's private, an implementation detail. The `wasmtime-environ/tests` directory is for integration tests, so we can't access private items. Thus, we have to have a tests submodule in the cache module. If you want to, I can move it to `wasmtime-environ/src/cache/tests.rs`.

Reply:
Yes, moving it to wasmtime-environ/src/cache/tests.rs sounds good.